location/motif.py

```diff
@@ -22,6 +27,8 @@
 import pandas as pd
 import numpy as np
 
+from location import utils
```

Comment:
I'd like to avoid circular imports — utils now imports motif and motif imports utils.

At the worst case, we should just implement `approx_home_location` here (given that we already have `get_primary_location`.